linq4j/src/main/java/org/apache/calcite/linq4j/tree/TryStatement.java

```diff
@@ -36,7 +37,17 @@ public TryStatement(Statement body, List<CatchBlock> catchBlocks,
   }
 
   @Override public Statement accept(Shuttle shuttle) {
-    return shuttle.visit(this);
+    shuttle = shuttle.preVisit(this);
+    Statement body1 = body.accept(shuttle);
+    List<CatchBlock> catchBlocks1 = new ArrayList<>();
+    for (CatchBlock cb: catchBlocks) {
+      Statement cbBody = cb.body.accept(shuttle);
+      catchBlocks1.add(
+          Expressions.catch_(cb.parameter, cbBody));
+    }
+    Statement fynally1 =
+        fynally == null ? null : fynally.accept(shuttle);
```

Comment:
Looks like this should be `finally`?

Reply:
@michaelmior Thanks for review! I addressed your comments. 
As "finally" is a keyword, I follow the variable name **fynally** in `TryStatement`.

Reply:
Of course. This slipped my mind when reviewing :)